theory-test-start-admission/lambda.js

```diff
@@ -1,5 +1,34 @@
+require('dotenv').config();
+
+const AdmissionDAO = require('theory-test-admission-dao');
 const logger = require('logger');
-const main = require('./src/main');
+
+exports.handler = (event, context, callback) => {
+
+	// log inbound event
+	logger.debug('Received event: ', JSON.stringify(event));
+	logger.info('Running start admission');
+	let response;
+	if (event.DrivingLicenceNumber && event.AdmissionId) {
+		const parameters = {
+			DrivingLicenceNumber: event.DrivingLicenceNumber,
+			HasBooking: event.HasBooking,
+			AdmissionId: event.AdmissionId
+		};
+
+		const admission = AdmissionDAO.createAdmissionDatabaseRecord(parameters);
+		AdmissionDAO.create(admission, (err, retVal) => {
+			if (!err) {
+				response = {
+					AdmissionId: retVal.AdmissionId,
+					valid: true,
```

Comment:
where has `valid` come from?

Reply:
I think this is the has-booking equivalient, removing